src/core/src/collection.rs

```diff
@@ -7,16 +7,24 @@ use crate::encodings::Idx;
 use crate::manifest::{Manifest, Record};
 use crate::prelude::*;
 use crate::storage::{FSStorage, InnerStorage, MemStorage, SigStore, ZipStorage};
+#[cfg(all(feature = "branchwater", not(target_arch = "wasm32")))]
+use crate::index::revindex::{ RevIndex, RevIndexOps };
 use crate::{Error, Result};
 
 #[cfg(feature = "parallel")]
 use rayon::prelude::*;
 
+/// a Manifest and Storage, combined. Can contain any collection of signatures.
+
+#[derive(Clone)]
 pub struct Collection {
```

Comment:
I was wondering if the docstring would connect to `Collection` even with the empty line.
```
cargo doc --open
```
it does show up =]